src/models/account.model.ts

```diff
@@ -0,0 +1,38 @@
+import Setting from './setting.model';
+import {
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  HasMany,
+  AllowNull,
+  Unique,
+} from 'sequelize-typescript';
+
+@Table
+export default class Account extends Model {
```

Comment:
By default id property is optional when extending the Model class, we should add the id property along with the @PrimaryKey decorator if the table contains id column

Reply:
[Fixed](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/src/models/account.model.ts#L20)